src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts

```diff
@@ -211,9 +215,12 @@ class FrameworkControllerTrainingService extends KubernetesTrainingService imple
                 //upload local files to azure storage
```

Comment:
local files are not clear, suggest to make it more clear, for example, -> "upload local files, including scripts for running the trial and configuration (e.g., hyperparameters) for the trial, to azure storage"

Reply:
fixed.